src/app/home/home.component.less

```diff
@@ -0,0 +1,18 @@
+.container-fluid {
```

Comment:
Please, don't override bootstrap classes. Use your own class.
Anyway, I have suggestions to improve style build in this app.
Create styles file in every module: moduleName/styles/moduleName.less
We can put all common styles ONLY related to this module to this file.

Also I suggest to move all forms common styles to forms.less file